release/stage-release.sh

```diff
@@ -9,13 +9,15 @@ if ! command -v releaser >/dev/null; then
 fi
 
 function usage() {
-  echo "Usage: ${0} VERSION" >&2
+  echo "Usage: ${0} VERSION GITHUB_TOKEN" >&2
   exit 1
 }
```

Comment:
Check if the appropriate env variable `CK8S_GITHUB_TOKEN` is set instead as that is better that to push users to input credentials as an argument which is less secure.